examples/wolfssl_client/main.c

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2006-2017 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
```

Comment:
The copyright note should be adapted to the same format as other RIOT files

Reply:
updated all notices